src/pages/Login/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import {
+  getAuth,
+  signInWithEmailAndPassword,
+  FacebookAuthProvider,
+  GoogleAuthProvider,
+  signInWithPopup,
+} from "firebase/auth";
+import { Button, Input } from "../../ components";
+import app from "../../services/firebase";
+import {
+  Alert,
+  Eye,
+  GoogleIcon,
+  FacebookIcon,
+  Hidden,
+} from "../../assets/index";
+
+import styles from "./styles.module.scss";
+import { changeUser } from "../../services/redux/userSlice";
+
+export const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [viewPassword, setViewPassword] = useState(false);
+  const [error, setError] = useState(false);
+  const auth = getAuth(app);
+  const dispatch = useDispatch();
+
+  const actionEmailAndPassword = async () => {
+    try {
+      const response = await signInWithEmailAndPassword(auth, email, password);
+      dispatch(changeUser(response));
+      setError(false);
+    } catch (error) {
+      setError(true);
+    }
+  };
+
+  const actionLoginFacebook = async () => {
+    try {
+      const provider = new FacebookAuthProvider();
+      const response = await signInWithPopup(auth, provider);
+      dispatch(changeUser(response));
+      setError(false);
+    } catch (error) {
+      setError(true);
+    }
+  };
+
+  const actionLoginGoogle = async () => {
+    try {
+      const provider = new GoogleAuthProvider();
+      const response = await signInWithPopup(auth, provider);
+      dispatch(changeUser(response));
+      setError(false);
+    } catch (error) {
+      setError(true);
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+      <main>
+        <span className={styles.title}>Colaborative Circuit Simulator</span>
+        <Input
+          onChange={(event) => setEmail(event.target.value)}
+          value={email}
+          placeholder="Email"
+        />
+        <div className={styles.passwordContainer}>
+          <Input
+            onChange={(event) => setPassword(event.target.value)}
+            value={password}
+            placeholder="Senha"
+            type={!viewPassword ? "password" : ""}
+          />
+          <button
+            onClick={() => setViewPassword(!viewPassword)}
+            className={styles.viewPassword}
+          >
+            {viewPassword ? (
```

Comment:
I suggested use only one img tag, and change the src by viewPassword flag
like this:
<img className={styles.imagePassword} src={viewPassword ? Hidden : Eye} alt=""/>
Oh, and please, add something to alt property, something like, eye, maybe